.github/workflows/package.yml

```
@@ -129,18 +134,109 @@ jobs:
+          tags: ${{ steps.base_meta.outputs.tags }}
+          labels: ${{ steps.base_meta.outputs.labels }}
           target: base_package
           build-args: |
             wrapper_file_name=${{steps.find_wrapper_file.outputs.WRAPPER_FILE_NAME}}
+
+      - name: jar meta
+        id: jar_meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{github.repository}}
+          flavor: |
+            latest=auto
+            suffix=jar,onlatest=true
+          tags: |
+            type=semver,pattern={{version}}
+            type=edge,branch=master
+            type=sha
+
+      - name: jar push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: ${{ startsWith(github.ref, 'refs/tags/*') || github.ref == 'refs/heads/master' }}
+          tags: ${{ steps.jar_meta.outputs.tags }}
+          labels: ${{ steps.jar_meta.outputs.labels }}
+          target: jar
+          build-args: |
+            wrapper_file_name=${{steps.find_wrapper_file.outputs.WRAPPER_FILE_NAME}}
+
+      - name: Darwin meta
+        id: Darwin_meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{github.repository}}
+          flavor: |
+            latest=auto
+            suffix=Darwin,onlatest=true
+          tags: |
+            type=semver,pattern={{version}}
+            type=edge,branch=master
+            type=sha
+
+      - name: Darwin push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: ${{ startsWith(github.ref, 'refs/tags/*') || github.ref == 'refs/heads/master' }}
+          tags: ${{ steps.Darwin_meta.outputs.tags }}
+          labels: ${{ steps.Darwin_meta.outputs.labels }}
+          target: Darwin
+          build-args: |
+            wrapper_file_name=${{steps.find_wrapper_file.outputs.WRAPPER_FILE_NAME}}
+
+      - name: Linux meta
+        id: Linux_meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{github.repository}}
+          flavor: |
+            latest=auto
+            suffix=Linux,onlatest=true
+          tags: |
+            type=semver,pattern={{version}}
+            type=edge,branch=master
+            type=sha
+
+      - name: Linux push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: ${{ startsWith(github.ref, 'refs/tags/*') || github.ref == 'refs/heads/master' }}
+          tags: ${{ steps.Linux_meta.outputs.tags }}
+          labels: ${{ steps.Linux_meta.outputs.labels }}
+          target: Linux
+          build-args: |
+            wrapper_file_name=${{steps.find_wrapper_file.outputs.WRAPPER_FILE_NAME}}
+
+      - name: Windows meta
+        id: Windows_meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{github.repository}}
+          flavor: |
+            latest=auto
+            suffix=Linux,onlatest=true
```

This doesn't seem to follow the pattern?

```suggestion
            suffix=Windows,onlatest=true
```